src/renderers/sketch/svg.js

```diff
@@ -10,9 +10,19 @@ const hatches = require('../../utility/misc/hatches.js');
 const hash = require('../../utility/misc/hash.js');
 const convert_units = require('../../utility/misc/convert-units.js');
 
+function convert_transform_matrix(_transform, scale) {
+  if (!_transform) return '';
+
+  const transform = _transform.slice();
+
+  if (scale && scale !== 1) {
+    const transformXIndex = 4;
+    const transformYIndex = 5;
+
+    transform[transformXIndex] /= scale;
+    transform[transformYIndex] /= scale;
+  }
 
-function convert_transform_matrix(transform) {
-  if (!transform) return '';
   return ` matrix(${transform.join(' ')})`;
```

Comment:
Leading space intended here?

Reply:
yes, as it gets appended to an HTML element attributes string